operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -0,0 +1,231 @@
+use core::slice;
+use futures::StreamExt;
+use gdal::{
+    raster::{Buffer, GdalType},
+    Driver,
+};
+use gdal_sys::{VSIFree, VSIGetMemFileBuffer};
+use geoengine_datatypes::{
+    primitives::{SpatialBounded, TimeInterval},
+    raster::{Blit, GeoTransform, Grid2D, GridShape2D, GridSize, Pixel, RasterTile2D},
+    spatial_reference::SpatialReference,
+};
+use std::{
+    convert::TryInto,
+    sync::mpsc::{Receiver, Sender},
+};
+use std::{ffi::CString, sync::mpsc};
+
+use crate::util::Result;
+use crate::{
+    engine::{QueryContext, QueryRectangle, RasterQueryProcessor},
+    error::Error,
+};
+
+pub async fn raster_stream_to_geotiff_bytes<T, C: QueryContext + 'static>(
+    processor: Box<dyn RasterQueryProcessor<RasterType = T>>,
+    query_rect: QueryRectangle,
+    query_ctx: C,
+    width: u32,
```

Comment:
why do you need width and hight? the query rectangle has a resolution

Reply:
9a47142823e2749a0e7a7c730b4de82ff969cda7

Reply:
great but i think it is not a good idea to hardcode the tile size 600 which was selected randomly

Reply:
this is just a test query rectangle